code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -130,3 +130,8 @@
 		for(var/obj/item/weapon/bomberman/dispenser in src)
 			if(dispenser.spam_bomb)
 				dispenser.attack_self(src)
+
+		if(get_active_hand() && istype(get_active_hand(),/obj/item/weapon/rcl))

```

Comment:
I was one of the people but not the only person to ask for it to be done this way.  Pressing the action key every time I make a step is ungainly.  It's easy to make a mistake and then you can't use the RCL to fix it, you have to take out wires, fix up the dodgy spot.. take out wire clippers.. remove all the fucking KNOTS you've accidentally made.. 

I think a continuous deployment is what it really needs to shine.